code/java/README.md

```diff
@@ -185,11 +118,13 @@ Class | Method | HTTP request | Description
 *ZakelijkeGerechtigdenApi* | [**getZakelijkGerechtigde**](docs/ZakelijkeGerechtigdenApi.md#getZakelijkGerechtigde) | **GET** /kadastraalonroerendezaken/{kadastraalonroerendezaakidentificatie}/zakelijkgerechtigden/{zakelijkgerechtigdeidentificatie} | 
 *ZakelijkeGerechtigdenApi* | [**getZakelijkGerechtigden**](docs/ZakelijkeGerechtigdenApi.md#getZakelijkGerechtigden) | **GET** /kadastraalonroerendezaken/{kadastraalonroerendezaakidentificatie}/zakelijkgerechtigden | 
 
+
 ## Documentation for Models
 
  - [Aantekening](docs/Aantekening.md)
  - [Adres](docs/Adres.md)
  - [BadRequestFoutbericht](docs/BadRequestFoutbericht.md)
+ - [BadRequestFoutberichtAllOf](docs/BadRequestFoutberichtAllOf.md)
```

Comment:
Dit ziet er vreemd uit - klopt dit met de `AllOf` erin? Misschien dat deze generator toch ander gedrag heeft.

Reply:
Ik kwam dit trouwens in je [eerste Java update](https://github.com/VNG-Realisatie/Haal-Centraal-BRK-bevragen/pull/565/commits/90e6057f4628d23bb48dde3931cadd6991289ca6) commit tegen. GitHub zegt dat dit nu 'outdated' is, maar ik kan niet goed zien met zoveel wijzigingen of dit nog steeds van toepassing is of niet.